examples/ex1p.cpp

```diff
@@ -225,13 +229,23 @@ int main(int argc, char *argv[])
    {
       if (UsesTensorBasis(fespace))
       {
-         prec = new OperatorJacobiSmoother(a, ess_tdof_list);
+#ifdef MFEM_USE_CEED
+         if (DeviceCanUseCeed() && algebraic_ceed)
+         {
+            prec = new AlgebraicCeedSolver(a, ess_tdof_list);
+         }
+         else
+#endif
```

Comment:
I will certainly do this if desired. But I want to be sure: doing this will mean copying a 300-line file with changes to 6 lines into a new example subdirectory. Is that what we want?

Reply:
This is a bit tricky because `AlgebraicCeedSolver` is not just a wrapper for something libCEED already does, so it is different in that regard from e.g. SUNDIALS or PETSc examples. That said, I also feel that keeping `ex1`+`ex1p` simple is better.

Maybe instead of creating a separate example, we can add this as an option in the multigrid example, `ex26`+`ex26p`, demonstrating another solver in MFEM. I'm not sure how well it will fit in there though.